torch/_dynamo/symbolic_convert.py

```diff
@@ -4166,6 +4166,33 @@ def speculate(self) -> SpeculationEntry:
             self.instructions[self.instruction_pointer - 1],
         )
 
+    def _make_frame_loc(
```

Comment:
ok nvm, this doesn't refactor any better than before - I think the previous one was better - sorry for the inconvenience